src/Tests/phpunit.xml

```diff
@@ -20,6 +20,6 @@
 	</filter>
 	<logging>
 		<!-- and this is where your report will be written -->
-		<log type="coverage-clover" target="./clover.xml"/>
+		<log type="coverage-clover" target="../../tests/output/clover.xml"/>
```

Comment:
I had to change the coverage output path here so that it aligns with the standard .drone.star file format

cc @phil-davis